### PR TITLE
Add Sections to .env.example

### PR DESCRIPTION
**Pull request by mike-moreau**
Breaking config settings into groups.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,46 +1,84 @@
...
+
 AWS_DEFAULT_REGION="us-east-1"
 AWS_S3_ACCESS_KEY_ID=""
 AWS_S3_SECRET_ACCESS_KEY=""
 AWS_S3_BUCKET=""
 
-CLOUDFRONT_URL=""
-CLOUDFRONT_DISTRIBUTION_ID=""
+# CLOUDFRONT_URL=""
+# CLOUDFRONT_DISTRIBUTION_ID=""
```
Comment (RandomJo) [inline .env.example]: @mikeatmostlys I'm curious why these CloudFront settings were commented out.

File: .env.example [hunk 2]
```diff
@@ -1,46 +1,84 @@
...
 
-RECAPTCHA_SITE_KEY=""
-RECAPTCHA_SECRET_KEY=""
+################################################
+# Mostly Serious QA and Support
+################################################
 
-MARKERIO_DESTINATION=""
+# MARKERIO_DESTINATION=""
+# SENTRY_DSN=""
```
Comment (RandomJo) [inline .env.example]: @mikeatmostlys We use Marker and Sentry by default on every project so these should probably not be commented it out. Seems like you're commenting out things that are optional, but these are standard.

File: .env.example (hunk 1 above)
Comment (mike-moreau) [reply .env.example]: Good catch, I didn't realize cloudfront was standard on every project at first. Will remove comments.

File: .env.example (hunk 2 above)
Comment (mike-moreau) [reply .env.example]: Good to know will remove comments.

**Pull request merged**